main/guides/governance/index.md

```diff
@@ -0,0 +1,263 @@
+    limitedCreatorFacet,
+  );
+  return harden({ publicFacet, creatorFacet: governorFacet });
+}
+```
+
+## Starting a Governed Contract via its Governor
+
+Changing parameter values is something only the electorate is authorized to do.
+To enforce this, a governed contract is started by a _contract governor_.
+The contract governor holds the fully-functional creator facet that includes
+the capability to change the parameters. The caller that starts the governor
+gets only the `limitedCreatorFacet`.
+
+<img src="./assets/gov-param-2.svg"
+  style="border: 2px solid" width="600" />
+
+_For clarity, some Zoe API details are ommitted from this figure._
+
+## Putting a question via an Election Manager
+
+An _ElectionManager_ is responsible for letting an appropriate party call `addQuestion()`. The `econCommitteeCharter.js` contract from `@agoric/inter-protocol` is sufficiently general to handle most forms of parameter
+governance, so we reuse it here. Swaparoo core eval deployment likewise
+starts this contract, asks it for invitations, and sends them to the voters:
+
+<img src="./assets/gov-param-3.svg"
+  style="border: 2px solid" width="600" />
+
+_For clarity, some Zoe API details are ommitted from this figure._
+
+The committee participant then instructs their smart wallet to
+redeem the charter invitation to get a capability to put a question using `VoteOnParamChange`,
+using `v0-accept-charter` to identify this offer:
+
+```js
+test.serial('Voter0 accepts charter, committee invitations', async t => {
+...
+  await victor.acceptCharterInvitation('v0-accept-charter');
+...
+});
+```
+
+<img src="./assets/gov-param-4-accept.svg"
+  style="border: 2px solid" width="600" />
+
+To exercises their `VoteOnParamChange` capability,
+the committee participant makes a _continuing invitation_,
+referring back to `v0-accept-charter` as `charterAcceptOfferId`:
+
+```js
+const makeVoter = (t, wallet, wellKnown) => {
+...
+    const putQuestion = async (offerId, params, deadline) => {
+    const instance = await wellKnown.instance[contractName]; // swaparoo instance handle
+    const path = { paramPath: { key: 'governedParams' } };
+
+    /** @type {import('@agoric/inter-protocol/src/econCommitteeCharter.js').ParamChangesOfferArgs} */
+    const offerArgs = harden({ deadline, params, instance, path });
+
+    /** @type {import('@agoric/smart-wallet/src/offers.js').OfferSpec} */
+    const offer = {
+        id: offerId,
+        invitationSpec: {
+        source: 'continuing',
+        previousOffer: NonNullish(charterAcceptOfferId),
+        invitationMakerName: 'VoteOnParamChange',
+        },
+        offerArgs,
+        proposal: {},
+    };
+    return doOffer(offer);
+    };
+};
+```
+
+The `offerArgs` include a deadline and details of the params to change:
+
+```js
+test.serial('vote to change swap fee', async t => {
+...
+  const targetFee = IST(50n, 100n); // 50 / 100 = 0.5 IST
+  const changes = { Fee: targetFee };
+...
+  const deadline = 2n;
```

Comment:
I presume `deadline` is `Timestamp` rather than `RelativeTime`.
```suggestion
  const deadline = 2n; // outside of tests, this would be unix epoch time seconds
```

Reply:
yes... also, timer docs are Coming Soon

 - #948